### PR TITLE
docs(mc): Add getting started documentation

### PR DESCRIPTION
**Pull request by k88hudson**
There are some people who need to work with our code base outside of our team, so I started on some general documentation for the system add-on:

### CODE REVIEW

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Activity Stream (System add-on)
+
+## Where to ask questions
+
+- Most of the core dev team can be found on the `#activity-stream` mozilla irc channel.
+  You can also direct message the core team (`dmose`, `emtwo`, `jkerim`, `k8hudson`, `Mardak`, `nanj`, `r1cky`, `ursula`)
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: Trying to misdirect? ;) Who is this `k8` with one 8!

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  or our manager (`tspurway`)
+- Our Mozilla slack channel is #activitystream
+- File issues/questions on Github: https://github.com/mozilla/activity-stream/issues
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: A sub note that we typically triage every Monday to give some expectation of turnaround.

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Testing in Firefox Nightly
+
+If you just want to see the current version of Activity Stream in Firefox, you can
+install [Firefox Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/)
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: Might as well link to https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly with the anchor

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+You will need to to clone Activity Stream to a local directory from the `master`
+branch of our Github repository: https://github.com/mozilla/activity-stream
+
+```
+git clone https://github.com/mozilla/activity-stream.git
+```
+
+### Mozilla Central
+You will need a local copy of Mozilla Central in a directory named `mozilla-central`.
+That directory should be a sibling of of your local `activity-stream` directory (like so):
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: just one "of" instead of two

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+for instructions about how to download and configure Mozilla Central if you have
+never done so before.
+
+## Source code
+
+The latest copy of the source code for Activity Stream can be found in the `system-addon/`
+subdirectory of `activity-stream`.
+
+The current version of Activity Stream that is shipped with Firefox nightly can
+be found in `mozilla-central/browser/extensions/activity-stream`.
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: Probably note that this code can be examined and edited for quick testing, but some files are generated, so it'll be better to go to the actual source.

File: docs/v2-system-addon/1.GETTING_STARTED.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+## Building
+
+1. Install required dependencies by running `npm install`.
+2. To build Activity Stream, run `npm run buildmc` from the root of the
+`activity-stream` directory. This will build the js and css files and copy them
+into the `browser/extensions/activity-stream` directory inside Mozilla Central.
+3. Build and run Firefox from the `mozilla-central` directory by running `./mach build && ./mach run`.
+
+## Continuous development
```
Comment (Mardak) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: A section on testing would be good for `npm run testmc` and how it has a larger set of tests than what gets run in `mozilla-central`.

File: docs/v2-system-addon/1.GETTING_STARTED.md (hunk 1 above)
Comment (k88hudson) [inline docs/v2-system-addon/1.GETTING_STARTED.md]: oooops lololol

**Comment by as-pine-proxy** [comment]
[Treeherder: 815c651e890d5a13e7926218d7b9b8eb845407a4](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=815c651e890d5a13e7926218d7b9b8eb845407a4)

**Pull request merged**